tests/ci/submitpytest.py

```diff
@@ -0,0 +1,177 @@
+"""
```

Comment:
do you guys think we should change the name of this file to explicitly mention that we are submitting a test through azureml?

Reply:
done